src/utilities/findUserWithFullName.js

```diff
@@ -0,0 +1,9 @@
+export function findUserWithFullName(usersArray, nameToFind) {
```

Comment:
This is not how the `findUserWithFullName` should work. Take another look at the example provided in the exercise and correct it.